app/views/devise/invitations/edit.html.haml

```diff
@@ -0,0 +1,20 @@
+.container
```

Comment:
Where do we use this?

Reply:
When the user clicks on the invitation link in the email message then we are redirected here for registration.